formatters/simplecov/simplecov.go

```diff
@@ -32,39 +32,16 @@ func (f *Formatter) Search(paths ...string) (string, error) {
 
 func (r Formatter) Format() (formatters.Report, error) {
 	rep, err := formatters.NewReport()
+
 	if err != nil {
 		return rep, err
 	}
 
-	jf, err := os.Open(r.Path)
-	if err != nil {
-		return rep, errors.WithStack(errors.Errorf("could not open coverage file %s", r.Path))
-	}
+	rep, err = jsonFormat(r,rep)
```

Comment:
I'm confused. Does this override what line 34 does?

Reply:
Do you mind taking a new look ? I don't think I'm following the the question sorry

Reply:
I might have something confused in my head, not seeing a problem here anymore 🤦